library/java/net/openid/appauth/AuthorizationManagementActivity.java

```diff
@@ -103,11 +104,18 @@
  *       destroyed, if necessary, such that AuthorizationManagementActivity is once again at the
  *       top of the back stack.
  *
- *     - Step S3: The pending intent provided to
- *       {@link AuthorizationService#performAuthorizationRequest} for completion of the
- *       authorization flow is invoked, providing the decoded {@link AuthorizationResponse} or
- *       {@link AuthorizationException} as appropriate. The AuthorizationManagementActivity
- *       finishes, removing itself from the back stack.
+ *     - Step S3a: If this activity was invoked via
```

Comment:
You reference some new steps here which are not in the diagram; could you add the new arrows to distinguish 3a / 3b?